css/style.css

```diff
@@ -507,33 +500,54 @@ address {
+.fl-theme-by .floe .floe-image-cloud           { background-image: url("../images/by-floe-05.png"); }
+.fl-theme-by .floe .floe-image-preferenceIcons { background-image: url("../images/by-floe-04.png"); }
+.fl-theme-by .floe .floe-image-videoPlayer     { background-image: url("../images/by-floe-03.png"); }
+
+.fl-theme-yb .floe .floe-image-UIO             { background-image: url("../images/yb-floe-02.png"); }
+.fl-theme-yb .floe .floe-image-exploreTool     { background-image: url("../images/yb-floe-01.png"); }
+.fl-theme-yb .floe .floe-image-cloud           { background-image: url("../images/yb-floe-05.png"); }
+.fl-theme-yb .floe .floe-image-preferenceIcons { background-image: url("../images/yb-floe-04.png"); }
+.fl-theme-yb .floe .floe-image-videoPlayer     { background-image: url("../images/yb-floe-03.png"); }
+
+.fl-theme-lgdg .floe .floe-image-UIO             { background-image: url("../images/lgdg-floe-02.png"); }
+.fl-theme-lgdg .floe .floe-image-exploreTool     { background-image: url("../images/lgdg-floe-01.png"); }
+.fl-theme-lgdg .floe .floe-image-cloud           { background-image: url("../images/lgdg-floe-05.png"); }
+.fl-theme-lgdg .floe .floe-image-preferenceIcons { background-image: url("../images/lgdg-floe-04.png"); }
+.fl-theme-lgdg .floe .floe-image-videoPlayer     { background-image: url("../images/lgdg-floe-03.png"); }
+
+.fl-theme-bw .floe .hewlett-logo-container svg, .fl-theme-wb .floe .hewlett-logo-container svg, .fl-theme-by .floe .hewlett-logo-container svg, .fl-theme-yb .floe .hewlett-logo-container svg, .fl-theme-lgdg .floe .hewlett-logo-container svg, .fl-theme-bw .floe .idrc-logo-container svg, .fl-theme-wb .floe .idrc-logo-container svg, .fl-theme-by .floe .idrc-logo-container svg, .fl-theme-yb .floe .idrc-logo-container svg, .fl-theme-lgdg .floe .idrc-logo-container svg {
+    border: 1px solid currentColor;
+}
+
+.fl-theme-bw .floe .hewlett-logo-container svg path, .fl-theme-wb .floe .hewlett-logo-container svg path, .fl-theme-by .floe .hewlett-logo-container svg path, .fl-theme-yb .floe .hewlett-logo-container svg path, .fl-theme-lgdg .floe .hewlett-logo-container svg path, .fl-theme-bw .floe .idrc-logo-container svg path, .fl-theme-wb .floe .idrc-logo-container svg path, .fl-theme-by .floe .idrc-logo-container svg path, .fl-theme-yb .floe .idrc-logo-container svg path, .fl-theme-lgdg .floe .idrc-logo-container svg path, .fl-theme-bw .floe .idrc-logo-container svg text, .fl-theme-wb .floe .idrc-logo-container svg text, .fl-theme-by .floe .idrc-logo-container svg text, .fl-theme-yb .floe .idrc-logo-container svg text, .fl-theme-lgdg .floe .idrc-logo-container svg text, .fl-theme-bw .floe .idrc-logo-container svg circle, .fl-theme-wb .floe .idrc-logo-container svg circle, .fl-theme-by .floe .idrc-logo-container svg circle, .fl-theme-yb .floe .idrc-logo-container svg circle, .fl-theme-lgdg .floe .idrc-logo-container svg circle {
```

Comment:
instead of having all the selectors on a single line, could you add line breaks after each comma. At this length I'm finding it hard to read and see what all the selectors are. Same for the other contrast theme related declaration above.